net-p2p/createtorrent/createtorrent-1.1.4-r1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools eutils
+
+DESCRIPTION="Create BitTorrent files easily"
+HOMEPAGE="http://www.createtorrent.com/"
+SRC_URI="http://www.createtorrent.com/${PN}-${PV}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
Is it "GPL-2" or "GPL-2+"?

Reply:
All the existing pages (BSD and SourceForge) refer to the license as GPL-2.  I should not change this, should I?

Reply:
"GPL-2+" means GPL-2 or later and "GPL-2" means GPL-2 only